app/components/insights-glance.js

```diff
@@ -0,0 +1,87 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { or } from '@ember/object/computed';
+import { format as d3format } from 'd3';
+
+export default Component.extend({
+  classNames: ['insights-glance'],
+  classNameBindings: ['isLoading:insights-glance--loading'],
+
+  isLoading: true,
+  isEmpty: false,
+
+  title: '',
+  statistic: '',
+
+  delta: 0,
+  deltaTitle: '',
+  deltaText: '',
+
+  labels: [],
+  values: [],
+  datasetTitle: 'Data',
+  centerline: null,
+
+  showPlaceholder: or('isLoading', 'isEmpty'),
+
+  // Chart component data
+  data: computed('values', 'labels', 'datasetTitle', function () {
```

Comment:
I think the property depends on the content of the arrays, so it should be `'values.[]', 'labels.[]'`

Reply:
Ahh cool, that makes sense! Change made, thanks 😄